dev.robiot.xclicker.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally it should be maintained in your upstream repo. It's not specific to Flathub.

Reply:
this the upstream repository should maintain and install this so that it's not needed here. It is not specific to Flathub.

Reply:
So I should move this file to https://github.com/robiot/xclicker and then download and install the file from there?

Reply:
for a future release is reasonable enough.